.github/workflows/zap_scan.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   zap_scan_ocp4:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Wasn't this ubuntu update in a previous PR?

Reply:
For a different workflow. I'm sneaking it in here.

Reply:
I will confess now that I approved this without testing it. Shame on me. Will play around with it now.

Reply:
Tested it out in dev - map seems to be working a-ok.